plugin/src/main/java/org/bonitasoft/plugin/MavenSessionExecutor.java

```diff
@@ -64,15 +75,43 @@ public BuildException(String message, Throwable cause) {
     /** The maven session */
     private MavenSession session;
 
+    /** Logger for Maven-style logging */
+    private Log log = new SystemStreamLog();
+
+    /** Project builder for parsing POM files */
+    private ProjectBuilder projectBuilder;
```

Comment:
Check if the ProjectBuilder is required to parse the POM to retrieve the dependencies